fossa/pr-check/action.yml

```diff
@@ -0,0 +1,81 @@
+name: pr check
+
+description: |
+  Check for new issues introduced by a pull request.
+
+  Note: Currently limited to license-related issues.
+
+inputs:
+  api-key:
+    description: The API key to access fossa.com
+    required: true
+  base-ref:
+    description: |
+      The ref (name) of the base branch in the context of a pull request.
+      Only for display purposes, not required.
+    required: false
+  base-revision:
+    description: |
+      The revision (commit SHA) of the base branch in the context of a pull request.
+    required: true
+  configuration-file:
+    description: Path to the FOSSA configuration file
+    required: false
+    default: .fossa.yml
+  path:
+    description: Path to the directory for scanning
+    default: .
+    required: false
+  project:
+    description: |
+      The project name.
+      Only for display purposes, not required.
+    default: main
+    required: false
+  revision:
+    description: |
+      The revision (commit sha) of the HEAD branch in the context of a pull request.
+    required: true
+
+runs:
+  using: composite
```

Comment:
I was wondering: Did you also consider using the 1st party [Github Action](https://github.com/fossas/fossa-action) provided by Fossa. They also seem to provide a [test-diff-version](https://github.com/fossas/fossa-action?tab=readme-ov-file#test-diff-revision) function which might be sufficient.

Reply:
Good question. Yes, I considered it, but it doesn't support all the parameters we use with the FOSSA CLI, and that was a blocker. This approach also gives us full control over the interface, which is especially useful when integrating multiple repositories as we do.

Creating these composite actions, and so an interface, doesn’t prevent us from using 1st party gha (behind the interface) in the future if it ends up being a better fit.